worlds/civ_6/Options.py

```diff
@@ -0,0 +1,105 @@
+    display_name = "Exclude Missable Boosts"
+
+
+class ResearchCostMultiplier(Choice):
+    """Multiplier for research cost of techs and civics, higher values make research more expensive. Cheap = 0.5x, Expensive = 1.5x. Default is 1. """
+    display_name = "Tech/Civic Cost Multiplier"
+    option_cheap = 0.5
+    option_default = 1
+    option_expensive = 1.5
+    default = 1
+
+
+class PreHintItems(Choice):
+    """Controls if/what items in the tech/civics trees are pre-hinted for the multiworld.\n
+    All : All items in the tech & civics trees are pre-hinted.\n
+    Progression items: Only locations in the trees containing progression items are pre-hinted.\n
+    No Junk: pre-hint the progression and useful items.\n
+    None: No items are pre-hinted.
+    """
+    display_name = "Tech/Civic Tree pre-hinted Items"
+    option_all = 0
+    option_progression_items = 1
+    option_no_junk = 2
+    option_none = 3
+    default = option_progression_items
+
+
+class HideItemNames(Toggle):
+    """Each Tech and Civic Location will have a title of 'Unrevealed' until its prereqs have been researched. Note that hints will still be precollected if that option is enabled."""
+    display_name = "Hide Item Names"
+
+
+class InGameFlagProgressionItems(DefaultOnToggle):
+    """If enabled, an advisor icon will be added to any location that contains a progression item"""
+    display_name = "Advisor Indicates Progression Items"
+
+
+class DeathLinkEffect(Choice):
+    """What happens when a unit dies. Default is Unit Killed.\n
+    Faith, and Gold will be decreased by the amount specified in 'Death Link Effect Percent'. \n
+    Era score is decreased by 1.\n
+    Any will select any of these options any time a death link is received."""
+    display_name = "Death Link Effect"
+    option_unit_killed = 0
+    option_era_score = 1
+    option_gold = 2
+    option_faith = 3
+    option_any = 4
+    option_any_except_era_score = 5
+    default = option_unit_killed
+
+
+class DeathLinkEffectPercent(Range):
+    """The percentage of the effect that will be applied. Only applicable for Gold and Faith effects. Default is 20%"""
+    display_name = "Death Link Effect Percent"
+    default = 20
+    range_start = 1
+    range_end = 100
+
+
+@dataclass
+class CivVIOptions(PerGameCommonOptions):
+    start_inventory_from_pool: StartInventoryPool
+    progression_style: ProgressionStyle
+    shuffle_goody_hut_rewards: ShuffleGoodyHuts
+    boostsanity: BoostSanity
+    exclude_missable_boosts: ExcludeMissableBoosts
+    research_cost_multiplier: ResearchCostMultiplier
+    pre_hint_items: PreHintItems
+    hide_item_names: HideItemNames
+    advisor_show_progression_items: InGameFlagProgressionItems
+    death_link_effect: DeathLinkEffect
+    death_link_effect_percent: DeathLinkEffectPercent
+    death_link: DeathLink
```

Comment:
```suggestion
    advisor_show_progression_items: InGameFlagProgressionItems
    death_link: DeathLink
    death_link_effect: DeathLinkEffect
    death_link_effect_percent: DeathLinkEffectPercent
```
Death link should probably be above its effect options

Reply:
💯